python/ray/dag/dag_node.py

```diff
@@ -82,9 +83,44 @@ def get_stable_uuid(self) -> str:
         """
         return self._stable_uuid
 
+    async def get_object_ref_from_last_execute(self, node_uuid: str):
```

Comment:
- looks like we are always getting all object refs anyway. 
- node_uuid is an implementation detail and it should not be part of the calling interfaces
- consider just returning the whole dictionary to simply

Reply:
Got it - would it be better if the interfaces used the actual node and accessed the uuid inside the function, or is it still preferable to return the whole dictionary?

Reply:
Should we hide UUID within `GraphVisualizer` as well? Changed to returning the whole dictionary, but it's still being used to get the intermediate outputs.